docs/general/ad-filtering/create-own-filters.md

```diff
@@ -803,9 +803,10 @@ Disables all generic [cosmetic rules](#cosmetic-rules) on pages that correspond
 * [`$permissions`](#permissions-modifier)
 * [`$redirect`](#redirect-modifier)
 * [`$redirect-rule`](#redirect-rule-modifier)
-* [`$replace`](#replace-modifier)
```

Comment:
oops

Reply:
Why? I moved them to be in alphabetical order:)

Reply:
I mean that I didn't sort them correctly :)